portfolio/src/main/webapp/charts_script.js

```diff
@@ -0,0 +1,31 @@
+google.charts.load('current', {'packages':['corechart']});
```

Comment:
Really minor comment, but maybe consider just naming the file `charts.js`? I like to avoid having redundant info in the filenames, and `.js` means script to me already.

Reply:
Agreed! Sounds great.